src/styles/blocks/clock.scss

```diff
@@ -0,0 +1,8 @@
+.clock {
+  position: relative;
+  width: 500px;
+  height: 500px;
+  margin: 0 auto;
```

Comment:
check this checklist point 
<img width="952" alt="image" src="https://user-images.githubusercontent.com/102411130/224640265-548426ae-6341-49c4-b83d-d9e947f8214c.png">